app/javascript/controllers/work_type_controller.js

```diff
@@ -10,13 +10,14 @@ export default class extends Controller {
   }
 
   change(event) {
-    this.areaTarget.hidden = false
-
     const type = event.target.value
-    const subtypes = document.subtypes[type].map((subtype)=> {
-      const id = subtype.replace(/\s/g, '_')
-      return this.templateTarget.innerHTML.replace(/SUBTYPE_LABEL/g, subtype).replace(/SUBTYPE_ID/g, id)
-    })
+    const subtypes = type === 'other' ? [this.otherTemplateTarget.innerHTML]
```

Comment:
Can you split this out into separate methods like "displayOtherSubtype" and "displaySubtypeOptions"?

Reply:
np

Reply:
@jcoyne done. have another look?